scex-core/src/main/scala/com/avsystem/scex/presentation/MathAttributes.scala

```diff
@@ -0,0 +1,113 @@
+      m.nextUp(_: Float) --> Attributes(documentation = "Returns the floating-point value adjacent to the `f` argument in the direction of positive infinity.")
+
+      m.pow(_: Double, _: Double) --> Attributes(documentation = "Returns the value of the `x` argument raised to the power of the the `y` argument. ")
+      m.rint _ --> Attributes(documentation = "Returns the closest floating point value that is equal to a mathematical integer for the `x` argument.")
+
+      m.round(_: Double) --> Attributes(documentation = "Returns the `x` argument rounded to the closest integer value.")
+      m.round(_: Float) --> Attributes(documentation = "Returns the `x` argument rounded to the closest integer value.")
+      m.round(_: Long) --> Attributes(documentation = "Returns the `x` argument rounded to the closest integer value.")
+
+      m.scalb(_: Double, _: Int) --> Attributes(documentation = "Returns `d` times 2 to the power of `scaleFactor` " + "rounded as if performed by a single correctly rounded floating-point multiply.")
+      m.scalb(_: Float, _: Int) --> Attributes(documentation = "Returns `f` times 2 to the power of `scaleFactor` " + "rounded as if performed by a single correctly rounded floating-point multiply.")
+
+      m.signum(_: Double) --> Attributes(documentation = "Returns the signum function of the `x` argument.")
+      m.signum(_: Float) --> Attributes(documentation = "Returns the signum function of the `x` argument.")
+      m.signum(_: Int) --> Attributes(documentation = "Returns the signum function of the `x` argument.")
+      m.signum(_: Long) --> Attributes(documentation = "Returns the signum function of the `x` argument.")
+
+      m.sin _ --> Attributes(documentation = "Returns the sine of the `x` argument.")
+      m.sinh _ --> Attributes(documentation = "Returns the hyperbolic sine of the `x` argument.")
+      m.sqrt _ --> Attributes(documentation = "Returns the square root of the `x` argument.")
+
+      m.subtractExact(_: Int, _: Int) --> Attributes(documentation = "Returns the difference of the `x` and `y` arguments.")
+      m.subtractExact(_: Long, _: Long) --> Attributes(documentation = "Returns the difference of the `x` and `y` arguments.")
+
+      m.tan _ --> Attributes(documentation = "Returns the tangent of the `x` argument.")
+      m.tanh _ --> Attributes(documentation = "Returns the hyperbolic tangent of the `x` argument.")
+
+      m.toDegrees _ --> Attributes(documentation = "Returns an angle measured in radians `x` converted to an approximately equivalent angle measured in degrees.")
+      m.toRadians _ --> Attributes(documentation = "Returns an angle measured in degrees `x` converted to an approximately equivalent angle measured in radians.")
+
+      m.ulp(_: Double) --> Attributes(documentation = "Returns the size of an ulp (unit in the last place) of the `x` argument.")
```

Comment:
I'd make it shorter, like: "Returns the size of a unit in the last place of the `x` argument." ; subsequent lines similarly